### ISSUE TITLE
Review performance

### ISSUE THREAD

**Issue by juanjoseortega-aircury** [curacultura/curacultura-website#88]
Review performance to improve it

**Comment by juanjoseortega-aircury** [comment]
Only things that can be improved is FCP ( First Contentful Paint) and LCP ( Largest Contentful Paint), the improvement for this maybe can take a lot of time to improve it only a little bit.
Also checked from the "Performance" tool, it shows that the FCP and L ( LCP ) it takes too low ms.
![Image](https://github.com/curacultura/curacultura-website/assets/128221593/de1ade73-a6ca-447f-8e4d-797b8fe1f6e0)

**Comment by cristobaldiaz-aircury** [comment]
Fair enough. This ticket can remain to investigate if there's something we can do to reduce the impact of FCPs and LCPs.

Moving back to TODO and reducing priority.
